tokens.json

```diff
@@ -1846,13 +1846,29 @@
         "chain": "eos"
     },
     {
-        "name": "Yup",
-        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUPX.png",
-        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUPX.png",
+        "name": "YUPX",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
         "symbol": "YUPX",
         "account": "yupyupxtoken",
         "chain": "eos"
     },
+    {
+        "name": "YUP",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "symbol": "YUP",
+        "account": "token.yup",
+        "chain": "eos"
+    },
+    {
+        "name": "YUPETH LP",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "symbol": "YUPETH LP",
```

Comment:
The symbol code seems wrong:
1. There's a white space
2. 8 character symbol codes is invalid